src/main/java/seedu/eylah/diettracker/model/food/Calories.java

```diff
@@ -25,6 +25,10 @@ public Calories(long calories) {
         this.value = calories;
     }
 
+    public long getValue() {
+        return value;
+    }
+
     /**
      * Returns true if a given string is a valid phone number.
      */
```

Comment:
Please edit this javadoc comment to reflect the actual usage.

Reply:
Alright thanks, changed!